docs/source/api/apollo-server.mdx

```diff
@@ -367,11 +367,11 @@ Defaults to `true` unless the `NODE_ENV` environment variable is `production` or
 
 ##### `logger`
 
-[`Logger`](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-types/src/index.ts#L178-L184)
+[`Logger`](https://github.com/apollographql/apollo-utils/tree/main/packages/logger)
```

Comment:
Maybe link to npmjs instead? (In all links in the PR.)

Reply:
[a69a065](https://github.com/apollographql/apollo-server/pull/6299/commits/a69a065a0db1ff8f770d9fdcddb2a217a0e49b82)